src/main/java/com/internet/shop/controllers/cart/DeleteProductFromCartController.java

```diff
@@ -0,0 +1,32 @@
+package com.internet.shop.controllers.cart;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.ProductService;
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/product/delete/cart")
+public class DeleteProductFromCartController extends HttpServlet {
```

Comment:
this is the next hw, let's remove this class from PR

Reply:
removed